app/icons/components/IcDragndrop2.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+function SvgIcDragndrop2(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
`function SvgIcDragndropCompact`?

Reply:
Thanks, I removed the older the older dnd icon.